packages/compass-icons/svgs/server-variant_E81F.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 25.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
Not sure if it will end up mattering of not, but this doesn't have a width/height defined. Will be interesting to see how this turns out!

I would suggest specifically testing this one.